### ISSUE TITLE
Add support to Sqlite database.

### ISSUE THREAD

**Issue by UlyssesAlves** [vanlooverenkoen/moor_db_viewer#50]

**Comment by vanlooverenkoen** [comment]
Hi very nice, I will try to implement it in the coming weeks

**Comment by UlyssesAlves** [comment]
> Hi very nice, I will try to implement it in the coming weeks

That would be great! Please let me know when it's available so that I can give it a try. I don't know if Github will notify me automatically, but if possible just mention me here so that I can know about it. :)

**Comment by vanlooverenkoen** [comment]
Yes I will do that. I will also ping you in the pull request itself.

**Comment by UlyssesAlves** [comment]
Great. Thank you.
